pages/api/users/[id]/index.js

```
@@ -25,6 +28,47 @@ export default async function handler(request, response) {
+        const fileName = `${Date.now()}-${file.originalFilename}`;
+        const newPath = path.join(process.cwd(), "public/uploads", fileName);
+        
+        // Move file to uploads directory
+        fs.renameSync(file.filepath, newPath);
+        
+        // Update image path in database
+        updateData.image = `/uploads/${fileName}`;
+      }
+
+      // Remove userId and id from update data (they're not user fields)
+      delete updateData.userId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)